templates/docs/patterns/notification.md

```diff
@@ -64,6 +64,16 @@ In cases where a notification sits inside another component, such as a table cel
 View example of the borderless notification pattern
 </a></div>
 
+#### Inset
+
+<span class="p-label--new">New</span>
+
+In cases where a notification message is directly relevant to what the user is doing on the page.
```

Comment:
> a notification message is directly relevant to what the user is doing on the page

In most our current use cases (without notification center/system) notifications are relevant to what the user is doing on the page, because they show based on user action.

We need to be more precise here.

And most importantly. This is visually extremely similar to 'borderless' variant. We need specific information when to use one over another.

Reply:
agreed, I've removed it.